content/hardware/05.pro-solutions/solutions-and-kits/edge-control/tutorials/smart-irrigation-system-v2/content.md

```diff
@@ -0,0 +1,568 @@
+---
+title: 'LoRaWAN® Irrigation System Using Arduino® Edge Control'
+description: "This application note describes how to control a four zones irrigation system using the Edge Control and the Arduino Cloud with LoRaWAN® connectivity"
+difficulty: intermediate
+tags:
+  - Irrigation System
+  - Arduino Edge Control
+  - Arduino MKR WAN 1310
+  - Solenoid Valve
+  - Arduino IoT Cloud
+  - Application Note
+  - WisGate Lite
+  - LoRaWAN®
+author: 'Christopher Mendez'
+libraries:
+  - name: Arduino_EdgeControl
+    url: https://github.com/arduino-libraries/Arduino_EdgeControl
+  - name: ArduinoJson
+    url: https://github.com/bblanchon/ArduinoJson
+  - name: Arduino_ConnectionsHandler
+    url: https://github.com/arduino-libraries/Arduino_ConnectionHandler
+  - name: ArduinoIoTCloud
+    url: https://github.com/arduino-libraries/ArduinoIoTCloud
+software:
+  - ide-v1
+  - ide-v2
+  - web-editor
+  - iot-cloud
+  - arduino-agent
+hardware:
+  - hardware/05.pro-solutions/boards/solutions-and-kits/edge-control
+  - hardware/05.pro-solutions/boards/solutions-and-kits/enclosure-kit
+  - hardware/01.mkr/01.boards/mkr-wan-1310
+---
+
+## Introduction
```

Comment:
Great introduction developing the intention of the application note

This is the idea! thanks!

Reply:
It is true that I normally prefer to have fewer paragraphs in the introduction.

But to be honest all of them are giving the reader nice and useful information and I don't find a way to reduce them so let's keep it as it is.

Reply:
Copied, thanks!